### ISSUE TITLE
fix: service info type should be non-configurable

### ISSUE THREAD

**Issue by mmalenic** [umccr/htsget-rs#283]
According to the [spec](https://samtools.github.io/hts-specs/htsget.html#ga4gh-service-info), the service info type field should probably be non-configurable, and set to the values for the htsget protocol:

```json
{"type": {"group": "org.ga4gh", "artifact": "htsget", "version": "x.y.z"}}
```

Currently it needs to be set manually by the config.

**Comment by brainstorm** [comment]
So you mean hardcode all those values and automatically fetch (burn as a const in the binary) the version attribute? Makes sense to me.

**Comment by mmalenic** [comment]
Yes. I think these values shouldn't change unless there is a new htsget version for example.
